.github/workflows/template-sync.yml

```diff
@@ -0,0 +1,18 @@
+name: template-sync
+
+on:
+  push:
+    branches:
+      - master
+      - chore/sync-target-repo
+
+jobs:
+  template_sync:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2 # important!
+      - uses: ahmadnassri/action-template-repository-sync@v1
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          dry-run: true
```

Comment:
Do you want to merge with dry-run set to true?

Reply:
Yeah, just for testing before merge it. I will also remove this branch from triggers branches.